### ISSUE TITLE
Warning for swap : Set swappiness tasks

### ISSUE THREAD

**Issue by roumano** [robertdebock/ansible-role-swap#1]
## Describe the bug
It's just a warning but should be fixed  :
  [WARNING]: The value "60" (type int) was converted to "'60'" (type string). If this does not look like what you expect, quote the entire value to ensure it does not change.

## Output

```
TASK [swap : Set swappiness] *****************************************************************************************************************************************************************
[WARNING]: The value "60" (type int) was converted to "'60'" (type string). If this does not look like what you expect, quote the entire value to ensure it does not change.
```

## Environment

- Control node OS: Ubuntu 22.04.1 LTS
- Control node Ansible version: ansible 2.10.8
- Managed node OS: Debian GNU/Linux 10 (buster)

## Fixing the issue
```
value: "{{ swap_swappiness }}"
```
should be replace by 
```
value: "{{ swap_swappiness | string }}"
```

Please consider [sponsoring me](https://github.com/sponsors/robertdebock).

**Comment by robertdebock** [comment]
PR is being tested now. Thanks!

**Comment by robertdebock** [comment]
This has been resolved in [version 2.1.3](https://github.com/robertdebock/ansible-role-swap/releases/tag/2.1.3).
